### PR TITLE
Fix typos and provide additional hints

### PR DESCRIPTION
**Pull request by seeebiii**
Hi, this is just a small PR to fix some typos and provide additional hints in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,9 +80,9 @@ npx jsii-release-maven [DIR]
...
 |`MAVEN_GPG_PRIVATE_KEY` or `MAVEN_GPG_PRIVATE_KEY_FILE` and `MAVEN_GPG_PRIVATE_KEY_PASSPHRASE`|Yes for Maven Central|GPG private key or file that includes it. This is used to sign your Maven packages. See instructions below|
-|`MAVEN_STAGING_PROFILE_ID`|Yes for Maven Central|Maven Central (sonatype) staging profile ID (e.g. 68a05363083174). Staging profile ID can be found **in the URL** of the "Releases" staging profile under "Staging Profiles" in https://oss.sonatype.org (e.g. `https://oss.sonatype.org/#stagingProfiles;11a33451234521`|
+|`MAVEN_STAGING_PROFILE_ID`|Yes for Maven Central|Maven Central (sonatype) staging profile ID (e.g. 68a05363083174). Staging profile ID can be found **in the URL** of the "Releases" staging profile under "Staging Profiles" in https://oss.sonatype.org if you are logged in (e.g. `https://oss.sonatype.org/#stagingProfiles;68a05363083174`).|
```
Comment (seeebiii) [inline README.md:85]: I though it's better to keep the same ID that was mentioned before to make it clear what is meant.

File: README.md [hunk 2]
```diff
@@ -106,9 +106,10 @@ Export and publish the public key:
 $ gpg -a --export > public.pem
 ```
 
-Go to https://keyserver.ubuntu.com/ and submit the public key
+Go to https://keyserver.ubuntu.com/ and submit the public key.
+You can use `cat public.pem` and copy/paste it into the "Submit Key" dialog.
```
Comment (seeebiii) [inline README.md:110]: `cat` works on Linux/Mac but I'm not sure if Windows works as well?

**Comment by seeebiii** [comment]
Should I fix the failing test regarding the copyright?!

**Comment by eladb** [comment]
@projen rebuild

**Review by eladb — APPROVED** [review]

**Pull request merged**